src/testing/builderbot/simulator/example_block.lua

```diff
@@ -0,0 +1,70 @@
+-- just as a reminder {black = 0, pink(magenta) = 1, orange = 2, green = 3, blue = 4}
+local color = {"magenta", "orange", "green", "blue"}
+color[0] = "black"
+
+local set_face_color = function(face, color)
```

Comment:
Is `set_face_color` used in any of the examples?

Reply:
Yes, line 49 to line 53. It is used to create a block with different colors on different faces

Reply:
Perhaps for this, I just add `set_face_color` to the control interface

Reply:
Actually that is not at all straight forward to do (which is probably why I didn't do it before). So let's just leave it as is.